src/Sylius/Component/Locale/Provider/TranslationLocaleProvider.php

```diff
@@ -9,7 +9,7 @@
 * file that was distributed with this source code.
 */
 
-namespace Sylius\Bundle\CoreBundle\Locale;
+namespace Sylius\Component\Locale\Provider;
 
 use Sylius\Component\Locale\Context\LocaleContextInterface;
 use Sylius\Component\Translation\Provider\LocaleProviderInterface;
```

Comment:
Actually this interface is from `Translation` component that is not required for `Locale` component. We can't move this class to `Locale` component before merging Translation component into Resource component.